examples/next/pages/ui/components/authenticator/i18n/index.page.tsx

```diff
@@ -0,0 +1,18 @@
+import { withAuthenticator } from '@aws-amplify/ui-react';
+import awsExports from '@environments/auth-with-username-no-attributes/src/aws-exports';
+import { Amplify, I18n } from 'aws-amplify';
+
+Amplify.configure(awsExports);
+
+I18n.setLanguage('fr');
```

Comment:
I already created an angular example that tests (but not documentation) for I18n on [`ui/components/authenticator/auth-with-translation`](https://github.com/aws-amplify/amplify-ui/blob/16013d22bca8247b27a1d77b9feac92df33fa660/examples/angular/src/pages/ui/components/authenticator/auth-with-translations/auth-with-translations.component.ts). Do you think we can combine these into one route?

Reply:
I also wrote a `.feature` file which I marked `@todo-react` and `@todo-vue`:

https://github.com/aws-amplify/amplify-ui/blob/16013d22bca8247b27a1d77b9feac92df33fa660/packages/e2e/features/ui/components/authenticator/auth-with-translations.feature#L7-L16

Reply:
One cool thing I did on my test is that I directly pulled in the expected translations from `@aws-amplify/ui` to do the translation testing like so: 

https://github.com/aws-amplify/amplify-ui/blob/16013d22bca8247b27a1d77b9feac92df33fa660/packages/e2e/cypress/integration/ui/components/authenticator/auth-with-translations/auth-with-translations.steps.ts#L4-L12

That way the test files is in sync with the translation file that we'll be developing over time. But this is more wordy and just testing for header text translated might be sufficient.

Reply:
I agree, so let's combine the best of both worlds with something like:

> `"Sign In" label is translated into "ja"`